internal/cli/server_run.go

```diff
@@ -205,12 +214,19 @@ func (c *ServerRunCommand) Run(args []string) int {
 		defer httpInsecureLn.Close()
 	}
 
+	telemetryEnabled := false
+	// If we have any export address configured, we should enable telemetry.
+	if c.flagTelemetryOpenCensusAgentAddr != "" || c.flagTelemetryDatadogTraceAddr != "" {
+		telemetryEnabled = true
```

Comment:
It'd be good to log via DEBUG that we enabled telemetry so users (and ourselves) can know

Reply:
We could also do it at the beginning of where we start to configure telemetry on line 331 🤔